README.md

```diff
@@ -1192,7 +1210,7 @@ service<http:Service> hello bind {port: 9090} {
 function doTweet() {
    // We can remove all the error handling here because we call
    // it asynchronously, don't want to get any output and
-   // don't care if it takes too long or fails
+   // don't care if it takes too long or fails.
```

Comment:
First letter not capitalized.

Reply:
It's the same sentence continuing from above line